packages/rtk-query-codegen-openapi/src/codegen.ts

```diff
@@ -67,30 +67,32 @@ export function generateCreateApiCall({
     const enhanceEndpointsObjectLiteralExpression = factory.createObjectLiteralExpression(
       [factory.createShorthandPropertyAssignment(factory.createIdentifier('addTagTypes'), undefined)],
       true
-    )
+    );
     return factory.createVariableStatement(
       undefined,
       factory.createVariableDeclarationList(
-        [factory.createVariableDeclaration(
-          factory.createIdentifier("injectedRtkApi"),
-          undefined,
-          undefined,
-          factory.createCallExpression(
-            factory.createPropertyAccessExpression(
-              factory.createCallExpression(
-                factory.createPropertyAccessExpression(
-                  factory.createIdentifier("api"),
-                  factory.createIdentifier("enhanceEndpoints")
+        [
+          factory.createVariableDeclaration(
+            factory.createIdentifier('injectedRtkApi'),
+            undefined,
+            undefined,
+            factory.createCallExpression(
+              factory.createPropertyAccessExpression(
+                factory.createCallExpression(
+                  factory.createPropertyAccessExpression(
+                    factory.createIdentifier('api'),
+                    factory.createIdentifier('enhanceEndpoints')
+                  ),
+                  undefined,
+                  [enhanceEndpointsObjectLiteralExpression]
                 ),
-                undefined,
-                [enhanceEndpointsObjectLiteralExpression]
+                factory.createIdentifier('injectEndpoints')
               ),
-              factory.createIdentifier("injectEndpoints")
-            ),
-            undefined,
-            [injectEndpointsObjectLiteralExpression]
-          )
-        )],
+              undefined,
+              [injectEndpointsObjectLiteralExpression]
+            )
+          ),
+        ],
```

Comment:
Looks like this file wasn't fully formatted with Prettier before? _(will check back in after CI finishes)_